doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
-Server will reference a server template, i.e, inherting all configuration of server template,
-Besides it may include additional information pim username, password, rack_name,position etc.
+Multiple instances are expected, one instance for each server.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| template_name | String | Yes | template referenced |
-| device_name | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
-| system_diskname | String | No | system disk name |
-| system_disksize | String | No | system disk size |
-| role | String | No | computer,controller,storage |
-| pim_ip_address | String | Yes | PIM ip address |
-| pim_netmask | String | Yes | PIM netmask |
-| pim_username | String | Yes | PIM user |
-| pim_password | String | Yes | PIM password |
-| hugepage_number | String | No | huge page quantity |
-| rack_name | String | Yes |  |
-| position | String | Yes |  |
-| remote_management_ip | String | Yes | remote management ip, e.g. iLO,iDRAC, BMC |
-| remote_user | String | Yes | remote user  |
-| remote_password | String | Yes | remote password |
-| nic_info | List | Yes | network interface information |
-
-<p align="center"><b>Table 6-3-3:</b> Server information.</p>
-
-<a name="6.3.4"></a>
-### 6.3.4 Software configuration definition
-It includes Virtualized Infrastructure Manager configurations,
+| NAME | String | Yes | Server name should be aligned with naming rule, this is the unique ID for each device to be referred for identify device |
+| VENDOR | String | Yes | Vendor of device |
+| SKU | String | Yes | The SKU of device, can be referred by other table NIC connection table, to identify slot-port mapping for device  |
+| MODEL | String | Yes | This is the model for different service type, this value is defined from design document of resource pool, example NC1, NC1-S |
+| SN | String | Yes | Serial Number |
+| RES_POOL | String | Yes | Resource pool name |
+| RACK | String | Yes | rack name where device located |
+| POS | String | Yes | the position of device in rack, like 2-3U,4-5U |
+| BMC_IP | String | Yes |  |
+| BMC_GATEWAY | String | Yes |  |
+| BMC_MASK | String | Yes |  |
+| BMC_SUBNET | String | Yes |  |
+| BMC_USR | String | Yes | BMC user |
+| BMC_PWD | String | Yes | BMC password |
+| INTERNAL_IP | String | Yes | It is an internal IP  configured and used by hardware integration tools, it will be removed after hardware integration verification |
+| INTERNAL_GATEWAY | String | Yes |  |
+| INTERNAL_MASK | String | Yes |  |
+| GROUP_NAME | String | Yes | the usage of server, Manage or Storaage or Service  |
+| BMC_PRE_CONFIGURED | String | Yes | YES or NO |
+| HW_REGION | String | Yes | hardware region that divided by design documents, like A area or B area |
```

Comment:
What does "design documents" refer to?

Reply:
I was referring pod design, like how many HA and how many nodes for each HA.

I have updated the description, could you help to check is it clear enough?